lib/modules/datasource/hackage/index.spec.ts

```diff
@@ -49,6 +49,7 @@ describe('modules/datasource/hackage/index', () => {
           {
             changelogUrl: baseUrl + 'package/base-4.20.0.1/changelog',
             version: '4.20.0.1',
+            isDeprecated: false,
```

Comment:
Also update the test to have a deprecated version

Reply:
Test added in [edb70c0](https://github.com/renovatebot/renovate/pull/33778/commits/edb70c0029d28539894734631597cc8775166a51)